test/resources/test_maskinporten.py

```diff
@@ -647,6 +647,7 @@ def test_create_client_key_to_aws(
     key = res.json()
     assert key == {
         "kid": "kid-1970-01-01-01-00-00",
+        "expires": "1970-04-01T00:00:00+00:00",
```

Comment:
Trenger vi en tilsvarende sjekk for annen expiration i `test_create_client_key_auto_rotate`?